conference/src/main/java/com/kyriba/conference/service/HallService.java

```diff
@@ -0,0 +1,19 @@
+package com.kyriba.conference.service;
+
+
+import com.kyriba.conference.api.dto.HallRequest;
+import com.kyriba.conference.domain.Hall;
+
+import java.util.List;
+
+
+public interface HallService
```

Comment:
No services are needed at that stage.

Reply:
yes, I just wanted to have a look how service should be mocked in tests